java-checks/src/test/files/checks/AssertionsInTestsCheck/Junit4.java

```diff
@@ -116,6 +119,14 @@ public void mockito_assertion_verify_no_more_interactions() {
     Mockito.verifyNoMoreInteractions(Mockito.mock(List.class));
   }
 
+  @Test
+  public void selenide_verify_should() {
+    Selenide.open("https://www.google.com/");
+    Selenide.$(By.name("q")).val("selenide").pressEnter();
+    Selenide.$$("#ires .g").shouldHave(CollectionCondition.sizeGreaterThan(1));
+    Selenide.$("#ires .g", 0).shouldHave(Condition.text("Selenide: concise UI tests in Java"));
```

Comment:
Each `should` assert should be in separate test cases, otherwise we cannot be sure which one(s) makes the rule compliant.